### PR TITLE
[MAD-2020] Adding placeholder

### PR DESCRIPTION
**Pull request by xavierportillaedo**
Here you can find Madrid 2020 event

### CODE REVIEW

File: data/events/2020-madrid.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+name: "2020-madrid" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2020" # The year of the event. Make sure it is in quotes.
+city: "Madrid" # The displayed city name of the event. Capitalize it.
+event_twitter: "DevOpsDaysMad" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
```
Comment (bridgetkromhout) [inline data/events/2020-madrid.yml]: Please register whatever twitter handle you plan to use - this one doesn't exist.

Comment (xavierportillaedo) [reply data/events/2020-madrid.yml]: Hello,
we are gonna create it after this merge and after creating the official emails.
BR

Comment (MikeRosTX) [reply data/events/2020-madrid.yml]: Create the Twitter handle first, so we can confirm it and then we will merge the PR.

**Comment by MikeRosTX** [comment]
We also need someone to send an email to **info@devopsdays.org** that includes all the organizer's **_emails_** with their **_first_** and **_last_** names.

**Comment by MikeRosTX** [comment]
Never mind, I found the list of emails for all organizers. Just need the Twitter handle to be created.

File: data/events/2020-madrid.yml (hunk 1 above)
Comment (xavierportillaedo) [reply data/events/2020-madrid.yml]: Hello,
We have crated the twitter account: https://twitter.com/DevOpsDaysMad
BR

**Pull request merged**